anyway/infographics_utils.py

```diff
@@ -1843,6 +1876,7 @@ def get_request_params(
 
     last_accident_date = get_latest_accident_date(table_obj=AccidentMarkerView, filters=None)
     # converting to datetime object to get the date
+    # TODO: handle last_accident_date is None
```

Comment:
There is no such case since we're talking about all of the accidents in Israel

Reply:
@atalyaalon , I wonder why do we need this calculation at all? Why not take today's date for end date?

Reply:
will use the same date filtering as implemented in get_date(). see below